replit/nixpkgs-replit
# Update typescript-language-server

# why

We're way behind, currently on 0.9.6 and 3.1.0 is already out!

# What changed

I updated it in `pkgs/node-packages/node-packages.json` and then ran `generate.sh`.
also updated `overlay.nix`

# Test plan

In an overlay repl

- `cd overlay && nix-build -A nodePackages.typescript-language-server`
- Edit replit.nix
```
{ pkgs, overlay ? import ./nixpkgs-replit/default.nix{ } }: {
    deps = [
      overlay.nodePackages.typescript-language-server
    ];
		env = {
			VERSION = overlay.replitPackages.version;
		};
}
```
- in shell `typescript-language-server --version`
- `3.1.0`
- Use in `.replit` as a language server (see nodejs repl)
- Works well!
